mlflow/tracking/client.py

```diff
@@ -694,6 +694,44 @@ def end_trace(
 
         self.end_span(request_id, root_span_id, outputs, attributes, status, end_time_ns)
 
+    def _log_trace(self, trace: Trace) -> str:
+        """
+        Log the complete Trace object to the backend store.
+
+        This is currently used for allowing integrators to log a trace object generated in a
+        different context to the backend store directly. The proper way to create a trace is
+        to use the `start_trace` and `end_trace` methods.
+
+        Args:
+            trace: The trace object to log.
+
+        Returns:
+            The request ID of the logged trace.
+        """
+        # Create trace info entry in the backend
+        # Note that the backend generates a new request ID for the trace. Currently there is
+        # no way to insert the trace with a specific request ID given by the user.
```

Comment:
```suggestion
        # NB: Since the backend API is used directly here, customization of request ID's 
        # are not possible with this internal API. A backend-generated ID will be generated
        # directly with this invocation.
```
A Nota Bene might read a bit better here